src/chunks/job-description-chunk/job-description-modal template/JobDescriptionModalChunk.tsx

```diff
@@ -133,16 +146,17 @@ const JobDescriptionModalChunk = (props: JobDescriptionCardProps) => {
                 <h1 className=" font-[600] text-16[px] leading-[25px] text-[#515B6F]">
                   Additional information
                 </h1>
-                <div className="border-[1px] border-[#D6DDEB]">
+
+                <div className="flex flex-col gap-0">
                   <textarea
                     placeholder="Add a cover letter or anything else you want to share"
                     className="font-[400] text-[16px] leading-[30px] text-[#515B6F99] font-epilogue 
-       border-[1px] h-[152px] border-[#D6DDEB] px-[16px]   w-full"
+       border-[1px] h-[152px] border-[#D6DDEB] p-[10px] outline-none   w-full"
```

Comment:
this #D6DDEB color, add it to the tailwind config file as an alias, and then use the alias instead here and everywhere else
and check for other colors that you're using, if they are currently in the tailwind config as aliases, then use them instead of writing out the hex code like that